src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -833,4 +833,23 @@ private static void addExecutablePermissionToDirectory(String filePath) {
                     + "'");
         }
     }
+
+    /**
+     * Update the tool if any latest version available.
+     *
+     * @param printStream stream which messages should be printed
+     */
+    public static void updateTool(PrintStream printStream) {
+        String version = ToolUtil.getCurrentToolsVersion();
```

Comment:
@suganyasuven 
This is a common logic used `bal update` command right?
https://github.com/ballerina-platform/ballerina-update-tool/blob/master/src/main/java/org/ballerinalang/command/cmd/UpdateToolCommand.java#L84

Shall we reuse this there as well ?

Reply:
Info logs are different for both scenarios.